src/openrct2/rct2/S6Importer.cpp

```diff
@@ -217,6 +217,8 @@ class S6Importer final : public IParkImporter
         gBankLoan        = _s6.current_loan;
         gParkFlags       = _s6.park_flags;
         gParkEntranceFee = _s6.park_entrance_fee;
+        // Force RCT2 to Unlock All Prices being false
+        gCheatsUnlockAllPrices = false;
```

Comment:
This does not really belong here. Rather, it should reset all cheats upon loading, like it does for MP already.